kubeflow/common/iap-ingress/Makefile

```diff
@@ -8,7 +8,7 @@ apply: iap-secret
 
 .PHONY: iap-secret
 iap-secret: check-iap
-	kubectl --context=$(KFCTXT) -n istio-system create secret generic kubeflow-oauth --from-literal=client_id=$(CLIENT_ID) --from-literal=client_secret=$(CLIENT_SECRET) --dry-run=client -o yaml | kubectl apply -f -
+	kubectl --context=$(KFCTXT) -n istio-system create secret generic kubeflow-oauth --from-literal=client_id=$(CLIENT_ID) --from-literal=client_secret=$(CLIENT_SECRET) --dry-run -o yaml | kubectl apply -f -
```

Comment:
this can be a little controversial, how can we be compatible with latest kubectl?
I think we should assume users use kubectl of version around 1.17 because kubectl and kubernetes version compatibility is within 2 minor version diff, see https://kubernetes.io/docs/setup/release/version-skew-policy/#kubectl.

So I think we can just make this change, WDUT?

Reply:
I think it is okay to apply this change. Thank you Yuan!